Formula/jsoncpp.rb

```diff
@@ -12,19 +12,16 @@ class Jsoncpp < Formula
     sha256 "5b4f345172c39853765b57eb0c7a706454031cf3a6da1c00a2b3d61244b20ada" => :sierra
   end
 
-  depends_on "cmake" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "python" => :build
```

Comment:
Why are `ninja` and `python` needed here? They are dependencies of `meson`, as far as I can see.

Reply:
This is the default when you run `brew create --meson`.

Reply:
I'll try to file a bug against that, then… I don't think our policy is to multiply indirect dependencies.

Reply:
But because it was auto-generated doesn't mean we should include it in the formula :)